bBridgeAPISDK/Common/HttpRequester.cs

```diff
@@ -51,7 +68,6 @@ public HttpRequester(Uri baseAdress, IAuthorizer authorizer = null)
                     {
                         if (responseTask.Result.IsSuccessStatusCode)
```

Comment:
Fixed to
responseTask.Result.StatusCode == HttpStatusCode.OK ||
					        responseTask.Result.StatusCode == HttpStatusCode.Accepted